src/assets/scripts/client/constants/themes/default/modules/terrain.js

```diff
@@ -0,0 +1,32 @@
+export const TERRAIN_THEME = {
+    /**
+     * Opacity of the lines on outer edges of terrain-elevated areas
+     *
+     * @memberof TERRAIN_THEME
+     * @property BORDER_OPACITY
+     */
+    BORDER_OPACITY: 1,
+
+    /**
+     * Opaxity of the fill color within terrain-elevated areas
+     *
+     * @memberof TERRAIN_THEME
+     * @property FILL_OPACITY
+     */
+    FILL_OPACITY: 0.1,
+
+    /**
+     * Colors for each elevation level (in feet), in RGB
+     *
+     * @memberof TERRAIN_THEME
+     * @property COLOR
+     */
+    COLOR: {
```

Comment:
this has been changed around quite a bit since this point (I hadn't really gotten to the terrain yet). However, I did leave it in place as an inline object, as it seemed to make a lot of sense to me to keep it that way.